engine-comparison/dispatcher.sh

```diff
@@ -66,6 +66,17 @@ build_benchmark_using() {
   mkdir $SEND_DIR
 
   cp ${BUILDING_DIR}/${BENCHMARK}-${FUZZING_ENGINE} $SEND_DIR
+
+  #TODO: make these gsutil cp?
```

Comment:
This is a true point, it would save a nontrivial amount of memory to have each runner grab the files it needs directly from the FTS directory; if for no other reason, I have this `cp` because `rsync` can only sync complete directories